next.config.js

```diff
@@ -26,6 +26,11 @@ const nextConfig = {
   experimental: {
     redirects() {
       return [
+        {
+          source: "/general",
+          destination: "/",
+          permanent: true,
+        },
         {
           source: "/general/:path*",
           destination: "/:path*",
```

Comment:
```suggestion
          source: "/general/:path+",
          destination: "/:path+",
```

https://github.com/pillarjs/path-to-regexp#zero-or-more 를 https://github.com/pillarjs/path-to-regexp#one-or-more 로 수정했습니다. 이렇게 하면 순서랑 관계없이 작동하겠네요.

Reply:
https://github.com/pillarjs/path-to-regexp#compile-reverse-path-to-regexp

destination `compile` 에 `undefined` 가 들어가서 처리를 못하고 에러가 발생하는 것 같아요.
빈 string 이 들어갈 줄 알았는데 `pathToRegexp` 돌려보니까 `undefined` 네요 😭